src/components/Block/BlockChain.tsx

```diff
@@ -11,8 +11,8 @@ export default function BlockChain(): JSX.Element {
 
   return (
     <div className="mx-3 row flex-nowrap overflow-auto">
-      {state.chain.map((block, i: number) => (
-        <Block details={block} key={`block${i}`} />
+      {state.chain.map((block) => (
+        <Block block={block} key={Math.random()} />
```

Comment:
For now this is acceptable and all functionality is correct with `Math.random()` since the **global** state is used.
However, this causes focus to be lost on the *block transaction* input when the user types which can lead to frustration - thus need to use a more specific key. However, when a more specific key was used, functionality broke - need to look into this further.

Reply:
Fixed in commit 90d577b

Problem was also that `defaultValue` input needed to be re-rendered properly in combination with local state management.